ivy/functional/backends/torch/experimental/manipulation.py

```diff
@@ -364,3 +364,19 @@ def unique_consecutive(
         inverse_indices,
         counts,
     )
+
+
+def trim_zeros(
+    filt: torch.Tensor,
```

Comment:
Regarding the typehint change, I was actually hinting to add something like `filt: Union[Sequence, torch.Tensor]` because you are later doing
```python
filt = torch.asarray(filt)
```
to account for non-tensor input, right? So we should expect `filt` to be something other than `torch.Tensor` as well.

Reply:
You are right, I have changed it.